### PR TITLE
Update v3.55.0

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #38 
Closes #37 
Closes #36 
Closes #35 

This PR

* Increases the version to `3.55.0` (I know that `4.0.0` is available but I'd need this and would update the feedstock afterwards)
* Adds me as a maintainer
* Removes bot merge again because it was failing. I have to bandwidth to update this manually.

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
   noarch: python
-  number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  number: 0
 
 requirements:
   host:
     - pip
-    - python
+    - python =2.7|>=3.4
```
Comment (BastianZim) [inline recipe/meta.yaml:20]: https://github.com/WoLpH/python-progressbar/blob/005d8a1f4816a450c7885253ce0b208ef1209126/setup.py#L102-L104

**Comment by BastianZim** [comment]
@marcelotrevisani When you have a second, would you mind having a look at this?

Strangely enough, here I didn't have the issue that I described in https://github.com/conda-incubator/grayskull/issues/288 – no idea why.

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by wolph** [comment]
Thank you for the help @BastianZim :)

**Comment by BastianZim** [comment]
Thanks for merging! :)

**Pull request merged**